src/components/Match/TeamTable.jsx

```diff
@@ -3,9 +3,16 @@ import PropTypes from 'prop-types';
 import { isRadiant, getTeamName } from 'utility';
 import Heading from 'components/Heading';
 import { IconRadiant, IconDire } from 'components/Icons';
+import { connect } from 'react-redux';
 import Table from 'components/Table';
 import PicksBans from './Overview/PicksBans'; // Displayed only on `Overview` page
 
+function highLightUser(loggedInId, players, side) {
+  const user = players.find(player => player.account_id === loggedInId);
+  const radiant = user && isRadiant(user.player_slot);
+  return user && (side === radiant) && (user.player_slot + 1 - (!radiant * 128));
```

Comment:
what is the third part?  not sure I understand it.

Reply:
it's to determine the row with the player slot.

128 + 1 - (1 * 128) = 1

Reply:
or if its radiant then

0 + 1 - (0 * 128) = 1